src/components/utils/filterTodos.ts

```diff
@@ -0,0 +1,27 @@
+import { SelectCategory } from '../../types/SelectCategory';
+import { Todo } from '../../types/Todo';
+
+export const filterTodos = (
+  todos: Todo[],
+  filters: { selectCaregory: SelectCategory; query: string },
```

Comment:
There is a typo in the parameter name `selectCaregory`. It should be `selectCategory` to match the naming convention used elsewhere in the application.